components/calendar/stories/calendar.stories.js

```diff
@@ -69,7 +86,8 @@ export default {
 			control: "boolean",
 		},
 		useDOWAbbrev: {
-			name: "Use 3 letter abbreviation for day of week",
+			name: "Days of the week",
+			description: "Use 3 letter abbreviation for day of week.",
```

Comment:
Since this boolean is specifically to change the days of the week from spelled out to abbreviated, I wonder if this update is actually less clear? Perhaps the title should be something like, "Use abbreviated weekdays"?